### PR TITLE
feat(CIS): Origin Post Quantum Encryption and Max HTTP Version

### PR DESCRIPTION
**Pull request by arjunchauhanibm**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: website/docs/r/cis_domain_settings.html.markdown [hunk 1]
```diff
@@ -189,12 +202,15 @@ Review the argument references that you can specify for your resource.
 - `true_client_ip_header` - (Optional, String) Supported values are `off` and `on`.
 - `waf` - (Optional, String) Enable a web application firewall (WAF). Supported values are `off` and `on`.
 - `websockets` - (Optional, String) Supported values are `off` and `on`.
+- `origin_max_http_version` - (Optional, String) Sets the highest HTTP version to use with origin. Supported values are `1` and `2`.
+- `origin_post_quantum_encryption` - (Optional, String) Weather to use Post-Quantum key agreement algorithms when connecting to the origin. Supported values are `off`, `preferred` and `supported`.
```
Comment (sheltons) [inline website/docs/r/cis_domain_settings.html.markdown:206]:
```suggestion
- `origin_post_quantum_encryption` - (Optional, String) Whether to use post-quantum key agreement algorithms when connecting to the origin. Supported values are `off`, `preferred`, and `supported`.
```

File: website/docs/r/cis_domain_settings.html.markdown [hunk 2]
```diff
@@ -189,12 +202,15 @@ Review the argument references that you can specify for your resource.
...
 
-**Note**
+### Note
 
 Extra settings are not implemented in this version of the provider.
- 
+
 ## Attribute reference
+
 In addition to all argument reference list, you can access the following attribute reference after your resource is created.
```
Comment (sheltons) [inline website/docs/r/cis_domain_settings.html.markdown:214]:
```suggestion
In addition to the argument reference list, you can access the following attribute reference after your resource is created.
```

**Review by sheltons — CHANGES_REQUESTED** [review]
Comments inline

File: ibm/service/cis/resource_ibm_cis_domain_settings.go [hunk 3]
```diff
@@ -349,6 +351,24 @@ func ResourceIBMCISSettings() *schema.Resource {
 						cisDomainSettingsCipherValidatorID),
 				},
 			},
+			cisDomainSettingsOriginMaxHTTPVersion: {
+				Type:        schema.TypeString,
+				Description: "origin_max_http_version setting",
```
Comment (hkantare) [inline ibm/service/cis/resource_ibm_cis_domain_settings.go:356]: Can we have some valid description

**Comment by hkantare** [comment]
Fix the conflicts

**Comment by hkantare** [comment]
Add respective testcase and test results

**Comment by arjunchauhanibm** [comment]
Test case 1.
origin_max_http_version = "1"
origin_post_quantum_encryption = "off"

image

image

**Comment by arjunchauhanibm** [comment]
Test case 2.
origin_max_http_version = "2"
origin_post_quantum_encryption = "preferred"

image

**Comment by arjunchauhanibm** [comment]
Test case 3.
origin_max_http_version = "2"
origin_post_quantum_encryption = "supported"

**Pull request merged**